### PR TITLE
Add Colombia adapter

### PR DESCRIPTION
**Pull request by ColombiaOnline**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

  Add Colombia adapter for Prebid .
Some test parameters for the bidder:
```
var adUnits = [{
    code: 'test-ad-div',
    sizes: [[300, 250]],
    bids: [{
    bidder: 'colombia',
      params: { 
        placementId: '44082'
       }
     }]
  }];
```
Our general contact email address is colombiaonline@timesinteret.in, and for anything related to this bidder, feel free to reach out to me at colombiaonline@timesinteret.in.

### CODE REVIEW

**Comment by jsnellbaker** [comment]
@ColombiaOnline When you get the chance please perform the following:
- rebase off of master (this should the circleci errrors)
- add unit tests for your adapter (please refer to this [page](http://prebid.org/dev-docs/bidder-adaptor.html#adding-unit-tests) for some additional details and a link to see how other adapters setup their unit tests).
- create a PR in the [docs repo](https://github.com/prebid/prebid.github.io) and add your bidder info under [here](https://github.com/prebid/prebid.github.io/tree/master/dev-docs/bidders).  Feel free to look at how other adapters setup their files if you need a reference (eg appnexus or rubicon).

**Comment by idettman** [comment]
CircleCI tests are failing

**Comment by ColombiaOnline** [comment]
@idettman 
We have fixed the circleci & LGTM errors. please continue to next process for merge.

**Comment by bretg** [comment]
docs PR at https://github.com/prebid/prebid.github.io/pull/937

**Comment by ColombiaOnline** [comment]
we are working on conflict and unit test cases.

File: package.json [hunk 1]
```diff
@@ -104,6 +104,7 @@
     "babel-plugin-transform-object-assign": "^6.22.0",
     "core-js": "^2.4.1",
     "gulp-sourcemaps": "^2.6.0",
-    "just-clone": "^1.0.2"
+    "just-clone": "^1.0.2",
+    "natives": "^1.1.4"
```
Comment (jsnellbaker) [inline package.json]: Why was this `natives` package added in?  It should be removed if there's no need for it.

**Comment by jsnellbaker** [comment]
See screenshot for CORS related error mentioned above:

screen shot 2018-09-12 at 9 37 50 am

**Comment by ColombiaOnline** [comment]
@jsnellbaker we have updated the code covering all the points pointed by you. Please review.

**Comment by jsnellbaker** [comment]
Hi @ColombiaOnline 

Thanks for making the various updates.  I tested the adapter again and found it working successfully now (no CORS errors).  

However, I noticed the `natives` package is still present in the `package.json` file.  If this is intentional, can you please elaborate on this change?  If not, please remove the extra package from the file and we should be good to merge.

Thanks.

**Comment by jsnellbaker** [comment]
@ColombiaOnline Actually, I forgot to mention something in the previous update.  In regards to the `spec.js` you added - in the various unit test functions, can you please convert the `,() => {` arrow syntax to the standard `, function() {` syntax?  

This styling change was recently implemented across the project in an effort to better comply with Mocha standards.  So we'd like new adapters to conform with the change.  Please see https://github.com/prebid/Prebid.js/issues/2987 for reference on the change.

**Comment by ColombiaOnline** [comment]
@jsnellbaker we have updated the code covering all the points pointed by you. Please review

**Comment by jsnellbaker** [comment]
@ColombiaOnline Thanks for the additional changes; LGTM.

**Pull request merged**